ext/standard/tests/array/array_diff_key_variation2.phpt

```diff
@@ -3,7 +3,7 @@ Test array_diff_key() function : usage variation - Passing unexpected values to
 --FILE--
 <?php
 /* Prototype  : array array_diff_key(array arr1, array arr2 [, array ...])
- * Description: Returns the entries of arr1 that have keys which are not present in any of the others arguments. 
+ * Description: Returns the entries of arr1 that have keys which are not present in any of the others arguments.
```

Comment:
Please do not submit unrelated whitespace changes! I fully agree that there shouldn't be any trailing whitespace, but automatically removing these may make diffs harder to read, and may cause merge conflicts. If at all, such whitespace changes should go to a separate commit/PR.

Reply:
@cmb69 Gonna revert these changes, sorry. We do need to find a way to trim all these whitespaces in tests, btw